src/components/Header/Header.jsx

```diff
@@ -1,11 +1,25 @@
-import React from "react";
-import './Header.scss'
+import React, { useState } from "react";
+import "./Header.scss";
 
-const Header = ({title}) =>
-    <header className="c-site-header">
-        <h1 className="c-site-header__title o-type__invisible">{title}</h1>
-        {/* look up component */}
-    </header>
+const Header = ({getNewLocation}) => {
+  const [city, setCity] = useState("");
+console.log({ city });
 
+  return (
+    <header className="header">
+      <input
```

Comment:
Best to wrap this in a `<form />` element so the user can submit by pressing enter while in an input